bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
Why do we want to do this? JS can easily parse that info out of the URL. This unnecessarily injects user-provided info into the template thus opening a potential XSS vector. It should be fine because the template should escape it, but it's still unnecessary.